tools/compile-coffee.py

```diff
@@ -10,6 +10,11 @@
   'C:/Program Files (x86)/nodejs',
   'C:/Program Files/nodejs',
 ] + os.environ['PATH'].split(os.pathsep)
+NIX_NODE_PATHs = [
+  '/usr/local/bin',
+  '/usr/bin',
+  '/bin',
+] + os.environ['PATH'].split(os.pathsep)
```

Comment:
We should put `$PATH` before custom locations, tools like `nvm` work by overwriting the `$PATH` variable.